### ISSUE TITLE
[BUG]: [ERROR] HTTPError 403: Forbidden

### ISSUE THREAD

**Issue by scopion1745** [anidl/multi-downloader-nx#651]
### Program version

4.6.3

### Operating System

Windows

### Type

GUI

### Service

Crunchyroll

### Command used

none

### Show ID

none

### Episode

none

### Console Output

```Shell
=== Multi Downloader NX GUI 4.6.0 ===

GUI server started on port 3000
[WS] Connection to public ws from '::1'
[WS] Connection from '::1'
[WS] Connection from '::1'
[ERROR] HTTPError 403: Forbidden
[ERROR] Just a moment...
[ERROR] Authentication failed!
[ERROR] HTTPError 403: Forbidden
[ERROR] Just a moment...
[ERROR] Authentication failed!
[ERROR] HTTPError 401: Unauthorized
[ERROR] Body: {"code":"accounts.get_profile.invalid_auth_token","context":[]}

[ERROR] Get profile failed!
```

### Additional Information

Message accours directly after launching the GUI. Issuing "Authentication" repeats the problem again and again.
Also tried a fresh pull of the repo, same results.

worked fine the last two days.
my cr account is still valid & i can stream via Browser.

tell me if you need any further logs or have any questions :).

**Comment by AnimeDL** [comment]
Pushed a commit that should fix it, if you would be willing to build it, test it, and give feedback, that would be nice, otherwise it will be available in the next release which will be after I test it enough

**Comment by AnimeDL** [comment]
Released version 4.7.0 which should fix the issue

**Comment by AnimeDL** [comment]
Going to close this since it was fixed
